package.json

```diff
@@ -28,18 +28,18 @@
     "@actions/github": "^1.1.0",
     "@actions/http-client": "^1.0.6",
     "@actions/io": "^1.0.2",
-    "@actions/tool-cache": "^1.3.3",
+    "@actions/tool-cache": "^1.5.3",
```

Comment:
This is in preview released from a branch.  As setup-python and setup-node finishes it's changes, we can feel confident in the api changes.